MathPortal/MathPortal/Source/Dialog/Task/TaskDetailsViewControllerTableViewCell.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  TaskDetailsViewControllerTableViewCell.swift
+//  MathPortal
+//
+//  Created by Petra Čačkov on 24/08/2019.
+//  Copyright © 2019 Petra Čačkov. All rights reserved.
+//
+
+import UIKit
+
+class TaskDetailsViewControllerTableViewCell: UITableViewCell {
```

Comment:
Needs rework:
- An added subview should also be removed at some point
- Setup cell should be split into 2 methods since the combination of both title and expression view is not possible
- Constraints should be used

Try:

```
class TaskDetailsViewControllerTableViewCell: UITableViewCell {

    @IBOutlet private var titleLabel: UILabel?
    @IBOutlet private var equationView: UIView?
    
    private var currentExpressionView: UIView?

    func setupCell(title: String?) {
        titleLabel?.text = title
        titleLabel?.backgroundColor = .clear
    }
    
    
    func setupCell(expressionView: UIView?) {
        currentExpressionView?.removeFromSuperview()
        currentExpressionView = nil
        
        guard let equationView = equationView else { return }
        
        if let expressionView = expressionView {
            equationView.addSubview(expressionView)
            expressionView.translatesAutoresizingMaskIntoConstraints = false
            equationView.addConstraint(NSLayoutConstraint(item: expressionView, attribute: .leading, relatedBy: .equal, toItem: equationView, attribute: .leading, multiplier: 1.0, constant: 10))
            equationView.addConstraint(NSLayoutConstraint(item: expressionView, attribute: .trailing, relatedBy: .equal, toItem: equationView, attribute: .trailing, multiplier: 1.0, constant: 10))
            equationView.addConstraint(NSLayoutConstraint(item: expressionView, attribute: .top, relatedBy: .equal, toItem: equationView, attribute: .top, multiplier: 1.0, constant: 5))
            equationView.addConstraint(NSLayoutConstraint(item: expressionView, attribute: .bottom, relatedBy: .equal, toItem: equationView, attribute: .bottom, multiplier: 1.0, constant: 5))
            currentExpressionView = expressionView
        }
    }
}
```